refinery/ui/source/js/commons/ctrls/download-files-ctrl.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+  'use strict';
+
+  angular
+  .module('refineryApp')
+  .controller('DownloadFilesCtrl', DownloadFilesCtrl);
+
+  DownloadFilesCtrl.$inject = [
+    '$window',
```

Comment:
https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#angularjs-and-javascript  

```Ordering Dependencies (alphabetize with groups)
3rd parties
Custom dependencies```